lib/constants.js

```diff
@@ -28,36 +32,47 @@ export const PRIZE_POOL_TYPE = Object.freeze({
 
 export const SUPPORTED_NETWORKS = [1, 3, 4, 31337, 1234]
 
+export const CTOKEN_UNDERLYING_TOKEN_DECIMALS = Object.freeze({
```

Comment:
We can avoid hard-coding this in the future and run `erc20Token.decimals()` using etherplex, etc to dynamically get the token's decimal precision, then fall back to 18 if the contract doesn't respond to that method (as it's not part of the ERC20 spec for some dumb reason)